mls-rs/Cargo.toml

```diff
@@ -30,6 +30,7 @@ out_of_order = ["private_message"]
 prior_epoch = []
 by_ref_proposal = []
 psk = []
+non_domain_separated_hpke_encrypt_decrypt = []
 x509 = ["mls-rs-core/x509", "dep:mls-rs-identity-x509"]
 rfc_compliant = ["private_message", "custom_proposal", "out_of_order", "psk", "x509", "prior_epoch", "by_ref_proposal", "mls-rs-core/rfc_compliant"]
 last_resort_key_package_ext = ["mls-rs-core/last_resort_key_package_ext"]
```

Comment:
You need to bump the version @emarteca else I can't publish it.

Reply:
Ah, thank you!